angular-chart.js

```diff
@@ -105,6 +114,23 @@
 
           if (usingExcanvas) window.G_vmlCanvasManager.initElement(elem[0]);
 
+          function aliasVar (fromName, toName) {
+            scope.$watch(fromName, function (newVal) {
+              if (typeof newVal == 'undefined') return;
```

Comment:
Should probably overwrite even when undefined otherwise it could have a slightly different behaviour.

Minor: always use `===` to avoid weird issues.